dlq-alert.tf

```diff
@@ -0,0 +1,30 @@
+// single alert to minify unnecessary cost because threshold used in here is minimal
+module "bulk-scan-exception-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
```

Comment:
let's enable first on AAT/Demo and then once working we can easily update this for prod